fail_modules.ml[4:9..4:9]: error
  │ 
4 │ let y = x
  │         ^
  Variable x not in scope
